### PR TITLE
Update Core link

### PR DESCRIPTION
**Pull request by chidozieononiwu**
Update link so that `azure-sdk-tools/eng/pipelines/templates/scripts/replace-relative-links.yml` can work on it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ For tutorials, samples, quick starts, and other documentation, go to [Azure for
...
 
 ### Client: November 2019 Releases
-New wave of packages that we are announcing as **GA** and several that are currently releasing in **preview**. These libraries follow the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet/guidelines/) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features at [Azure.Core](/sdk/core/Azure.Core/README.md).
+New wave of packages that we are announcing as **GA** and several that are currently releasing in **preview**. These libraries follow the [Azure SDK Design Guidelines for .NET](https://azure.github.io/azure-sdk/dotnet/guidelines/) and share a number of core features such as HTTP retries, logging, transport protocols, authentication protocols, etc., so that once you learn how to use these features in one client library, you will know how to use them in other client libraries. You can learn about these shared features at [Azure.Core](sdk/core/Azure.Core/README.md).
```
Comment (weshaggard) [inline README.md]: I thought @JimSuplizio updated our relative link parser to handle ones that begin with a `/`. In either case I'm fine with this change but it seems like we should support the leading `/` if markdown supports it.

**Review by weshaggard — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (chidozieononiwu) [reply README.md]: I will look into that.

**Comment by weshaggard** [comment]
Can you please run a doc index update as it looks like the link is still broken http://azure.github.io/azure-sdk-for-net/.

**Comment by chidozieononiwu** [comment]
> Can you please run a doc index update as it looks like the link is still broken http://azure.github.io/azure-sdk-for-net/.

Fixed Now.

**Pull request merged**